src/screens/Login/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { isEmpty } from 'lodash';
+import validator from 'validator';
+
+import AlertWarning from 'components/AlertWarning';
+import Button from 'components/Button';
+import WarningIcon from 'components/Icon/WarningIcon';
+import Input from 'components/Input';
+import AuthLayout from 'components/Layout/Auth';
+
+const LoginScreen = (): JSX.Element => {
+  const { t } = useTranslation();
+  const [formInput, setformInput] = useState({ email: '', password: '' });
+  const [errorMessage, setErrorMessage] = useState<string[]>([]);
+  const [formLoading, setFormLoading] = useState(false);
+  const validateInput = (input: { email: string; password: string }) => {
```

Comment:
Would you mind if you could add a line break between any function that spans multiple lines? it makes it easier to read 🙏 

Same with the `handleSubmit` and `handleChange`

It's part of our conventions in [Elixir](https://nimblehq.co/compass/development/code-conventions/elixir/#convention-23), but it works great in other languages too :smile:

```suggestion
  const [formLoading, setFormLoading] = useState(false);

  const validateInput = (input: { email: string; password: string }) => {
```

Reply:
Sure added in https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/8172c6c9c8c48e41c595245d1a91cef8b76650c7